encode_utils/connection.py

```diff
@@ -1118,19 +1118,20 @@ def patch(self, payload, raise_403=True, extend_array_values=True):
             for key in payload:
                 if isinstance(payload[key], list):
                     val = payload[key]
-                    if not isinstance(val, str):
-                        # Currently only able to work with arrays of strings. 
-                        continue
                     val.extend(rec_json.get(key, []))
                     # I use rec_json.get(key,[]) above because in a GET request,
                     # not all props are pulled back when they are empty.
                     # For ex, in a file object, if the controlled_by prop isn't set, then
                     # it won't be in the response.
- 
+
                     ## CHECK FOR DUPLICATES: Be careful as some can be tricky, i.e.
-                    # ['/documents/id1', 'id1'] 
-                    # such a duplicate should be identified and removed, leaving us with ["id1"]. 
-                    payload[key] = eup.remove_duplicate_associations(val)
+                    # ['/documents/id1', 'id1']
+                    # such a duplicate should be identified and removed, leaving us with ["id1"].
+                    # Currently only able to work with arrays of strings.
```

Comment:
Could you change this comment to reflect that dicts are now supported?

Reply:
Made the suggested changes.